plasma_framework/contracts/src/exits/payment/PaymentExitGame.sol

```diff
@@ -30,8 +30,7 @@ contract PaymentExitGame is IExitProcessor, OnlyFromAddress, PaymentStandardExit
         require(args.spendingConditionRegistry.owner() == address(0), "Spending condition registry ownership needs to be renounced");
     }
 
-    function init() public{
-        require(msg.sender == paymentExitGameArgs.framework.getMaintainer(), "Only Maintainer can perform this action");
+    function init() public onlyFrom(address(paymentExitGameArgs.framework.getMaintainer())) {
```

Comment:
The address() can be removed here
`onlyFrom(paymentExitGameArgs.framework.getMaintainer())`